meson.build

```diff
@@ -99,6 +99,9 @@ if git.found()
     # Now let's check if the working directory is clean.
     if run_command(git, 'diff-index', '--quiet', 'HEAD').returncode() == 0
       short_rev = r.stdout().strip()
+      if run_command(git, 'describe', '--exact-match', '--tags').returncode() == 0
```

Comment:
This line doesn't seem to compare that the tag is indeed at the same review id?

Reply:
Indeed, it isn't comparing anything - the checks are that if the tree is clean at the current commit (previous check) and that the current commit matches a tag (this check). Should I add a comment about it?